### PR TITLE
update homework submission based on github lesson, new workflow

### PR DESCRIPTION
**Pull request by bgveenstra**

### CODE REVIEW

**Comment by Nick-Brennan** [comment]
Looks good to me.  :)

File: how-to/homework-submission.md [hunk 1]
```diff
@@ -1,49 +1,71 @@
 #Instructions for Submitting Homework
 
-### Dawn/Dusk Exercise Submission
+Homework and labs will each have their own repository.  These repositories will usually live in the `sf-wdi-27-28` GitHub organization.
```
Comment (nathanallen) [inline how-to/homework-submission.md]: may need to omit `sf-wdi-27-28` to future-proof this

File: how-to/homework-submission.md [hunk 2]
```diff
@@ -1,49 +1,71 @@
...
+
+
+  ```zsh
+  ➜  cd ~/wdi
+  ➜  git clone <clone-url>
+  ```
+
+5. You should now have a copy of the repo inside your `~/wdi` directory! Change directories into the repo you cloned, and complete the tasks from the homework instructions. 
+
+1. Remember to commit your local changes as you go, and check `git status`!
```
Comment (tgaff) [inline how-to/homework-submission.md]: Should `git status` (and its output) be better integrated into this code block?

File: how-to/homework-submission.md [hunk 3]
```diff
@@ -1,49 +1,71 @@
...
-2. Next, `git push` your updates to your fork of the homework repo on github.
+7. Once you're done with the assignment and have committed and pushed ALL of your changes to GitHub, it's time to make a pull request back to the original repo. Go to your forked copy of the repo on GitHub, and click the "Pull Request" button.
+
+  ![make new pull request button screenshot](https://cloud.githubusercontent.com/assets/3010270/13687969/7ed9d38c-e6d3-11e5-9510-ac0fbf1f22e9.png)
+
+8. GitHub takes you to a new view and asks if you want to create the pull request. Add this information to the pull request:
+
+	Title: w01 d01
+
+	Description:
```
Comment (tgaff) [inline how-to/homework-submission.md]: The formatting here didn't work out

**Pull request merged**